docs/bigquery-usage.rst

```diff
@@ -397,13 +386,17 @@ the job locally:
 .. doctest::
 
    >>> from gcloud import bigquery
+   >>> from gcloud.bigquery import SchemaField
    >>> client = bigquery.Client()
    >>> table = dataset.table(name='person_ages')
-   >>> job = table.load_from_storage(bucket_name='bucket-name',
-   ...                               object_name_glob='object-prefix*',
-   ...                               source_format='CSV',
-   ...                               skip_leading_rows=1,
-   ...                               write_disposition='truncate')
+   >>> table.schema = [
```

Comment:
Yeah I don't have a sense for that one. Worth mentioning in a comment?

It seems someone has asked this on StackOverflow and gotten an [answer][1] like the one I suggested.

[1]: http://stackoverflow.com/a/21868338/1068170